### ISSUE TITLE
Some character sequences get merged

### ISSUE THREAD

**Issue by Pantalaim0n** [adobe-fonts/source-code-pro#192]
I've noticed some weird behaviour when typing a certain sequence of characters. They get squeezed to the width of one single character.

It happens when I type lowercase L, U+00B7 (middle dot), lowercase L
Using both uppercase L also shows this issue.

I began to notice it when I enabled 'Show non-printable characters' in Netbeans, which uses U+00B7 as a space character.
![merged_netbeans](https://user-images.githubusercontent.com/25281933/37758283-ee06e176-2daf-11e8-95f9-9d1ac5903165.jpg)

I could also reproduce this issue in Notepad:
![merged_notepad](https://user-images.githubusercontent.com/25281933/37758309-03b42e48-2db0-11e8-8d03-ca75d4b9a683.jpg)

I haven't found any other sequence that behaves the same.

**Comment by frankrolf** [comment]
This is a character sequence common in Catalan (e.g. `paral·lel`), and the behavior is triggered here:
https://github.com/adobe-fonts/source-code-pro/blob/b47764205aa316b2c172dc37921526dc45153e0a/Roman/familyGSUB.fea#L210-L211

It seems to be a bug in Netbeans to merge cross layer of printable characters with non-printable characters by using an OpenType feature across them.

**Comment by moyogo** [comment]
That `sub l periodcentered l by lcat;` and the one for Lcat are in `lookup GLYPH_COMPOSITION_LATIN_NONCONTEXTUAL` which is in `feature ccmp` and in DFLT dflt.
The ccmp feature is on by default, and this substitution is on in all languages.

This seems like Netbeans is just applying what is defined in the font.

**Comment by moyogo** [comment]
While Netbeans shouldn’t use the U+00B7 for displaying the space character to avoid the issue here, it’s a bit odd to expect that. One may use U+00B7 between two Ls and not expect a Catalan geminated L in some cases.

**Comment by frankrolf** [comment]
> This seems like Netbeans is just applying what is defined in the font.
That is correct, but Netbeans is also effectively breaking any other sequence-based OT feature through inserting U+00B7 into the character sequence.

What would you suggest? I can see the pragmatic approach (`ccmp`) being more accessible to Catalan users, rather than hiding the feature behind language tagging, which we’ve done for other fonts.

**Comment by escobera** [comment]
This issue is happening with VS Code as well
![image](https://user-images.githubusercontent.com/9344/97465939-55db9080-1921-11eb-8336-5b2471575110.png)

**Comment by frankrolf** [comment]
Can you bring that up with VS Code?

**Comment by escobera** [comment]
Someone already did here: https://github.com/microsoft/vscode/issues/106583

Thanks
